### ISSUE TITLE
Checkov fails to parse tf.json sources and returns an error to the console

### ISSUE THREAD

**Issue by marcoferrer** [bridgecrewio/checkov#807]
**Describe the bug**
When executing checkov against a directory with `tf.json` sources, an error is produced with the following message
```bash
       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 1.0.716 

terraform scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 0, Parsing errors: 1

Error parsing file /Users/marco.ferrer/my-project/cdk.tf.json

```
The contents of `cdk.tf.json` are as follows
```json
{
  "variable": {
    "environment": {
      "type": "string"
    },
    "aws_region": {
      "default": "us-east-1",
      "type": "string"
    },
    "aws_profile": {
      "type": "string"
    }
  },
  "terraform": {
    "required_providers": {
      "aws": {
        "version": "~> 2.70.0",
        "source": "aws"
      }
    }
  },
  "provider": {
    "aws": [
      {
        "profile": "${var.aws_profile}",
        "region": "${var.aws_region}",
        "alias": "default"
      },
      {
        "profile": "external",
        "region": "us-west-1",
        "skip_requesting_account_id": true,
        "alias": "external"
      }
    ]
  },
  "resource": {
    "aws_secretsmanager_secret": {
      "local-secret": {
        "name": "internal-secret",
        "provider": "aws.default"
      },
      "external-secret": {
        "name": "external-secret",
        "provider": "aws.external"
      }
    }
  }
}
``` 

**To Reproduce**
1. Create a new directory
2. Within this directory, create a new file with the suffix `.tf.json`
3. Populate the file with the json example from above or any other valid `tf.json` definition
4. Execute `checkov -d ./path/to/your/new/directory`
5. Review cli output for the parsing error

**Expected behavior**
The expected result is for the modules sources to be properly parsed and the built-in checkov rules asserted over the content 
 of the `tf.json` sources.

**Desktop (please complete the following information):**
 - OS: macOSX 10.15.5
 - Checkov Version 1.0.716 and earlier

**Comment by robeden** [comment]
I'm looking at this...

**Comment by robeden** [comment]
I'm not 100% sure everything will work perfectly as that json data looks different than I would expect, but the above PR at least works around the first error.

**Comment by marcoferrer** [comment]
The json is output from the official terraform-cdk tool. Is there something specifically that seems off with it? I can try to relay the concerns. Also as a side note, this example ommits the metadata generated by the tool which follows the `tf.json` spec. Here is the output in its original state which also fails to parse in checkov.

```json
{
  "//": {
    "metadata": {
      "version": "0.1.0",
      "stackName": "module-a"
    }
  },
  "variable": {
    "environment": {
      "type": "string"
    },
    "aws_region": {
      "default": "us-east-1",
      "type": "string"
    },
    "aws_profile": {
      "type": "string"
    }
  },
  "terraform": {
    "required_providers": {
      "aws": {
        "version": "~> 2.70.0",
        "source": "aws"
      }
    }
  },
  "provider": {
    "aws": [
      {
        "profile": "${var.aws_profile}",
        "region": "${var.aws_region}",
        "alias": "default"
      },
      {
        "profile": "external",
        "region": "us-west-1",
        "skip_requesting_account_id": true,
        "alias": "external"
      }
    ]
  },
  "resource": {
    "aws_secretsmanager_secret": {
      "local-secret": {
        "name": "internal-secret",
        "provider": "aws.default",
        "//": {
          "metadata": {
            "path": "module-a/local-secret",
            "uniqueId": "local-secret",
            "stackTrace": [
              "new TerraformElement (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/cdktf/lib/terraform-element.js:19:28)",
              "new TerraformResource (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/cdktf/lib/terraform-resource.js:15:9)",
              "new SecretsmanagerSecret (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/aws/providers/aws/secretsmanager-secret.js:21:9)",
              "/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3589:49",
              "Kernel._wrapSandboxCode (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:4073:16)",
              "Kernel._create (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3589:26)",
              "Kernel.create (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3324:21)",
              "KernelHost.processRequest (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13191:28)",
              "KernelHost.run (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13129:14)",
              "Immediate._onImmediate (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13132:37)",
              "processImmediate (internal/timers.js:456:21)"
            ]
          }
        }
      },
      "external-secret": {
        "name": "external-secret",
        "provider": "aws.external",
        "//": {
          "metadata": {
            "path": "module-a/external-secret",
            "uniqueId": "external-secret",
            "stackTrace": [
              "new TerraformElement (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/cdktf/lib/terraform-element.js:19:28)",
              "new TerraformResource (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/cdktf/lib/terraform-resource.js:15:9)",
              "new SecretsmanagerSecret (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-kernel-SYQiRC/node_modules/aws/providers/aws/secretsmanager-secret.js:21:9)",
              "/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3589:49",
              "Kernel._wrapSandboxCode (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:4073:16)",
              "Kernel._create (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3589:26)",
              "Kernel.create (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:3324:21)",
              "KernelHost.processRequest (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13191:28)",
              "KernelHost.run (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13129:14)",
              "Immediate._onImmediate (/private/var/folders/rb/lzny6mx11wbgq3s1yn7yh7g00000gn/T/jsii-java-runtime15937660924015765477/jsii-runtime.js:13132:37)",
              "processImmediate (internal/timers.js:456:21)"
            ]
          }
        }
      }
    }
  }
}
```

**Comment by robeden** [comment]
In surprised some values aren’t contained in arrays. But, I doubt there’s anything wrong with the tool. I expect more that how the data is represented can be flexible. I didn’t mean by the statement that you need to do anything other than know that there might be some follow-up issues. Handling of TF in .json files is new and the path is not yet well trodden.

**Comment by marcoferrer** [comment]
Oh ok, I understand. Thanks again for the help. Yeah, I'm currently working my way through testing out our workflows but through json instead of HCl. I sure there will be some hiccups along the way so I'll do my best to bring up any issues I might run into.

**Comment by nimrodkor** [comment]
From looking at the structure of the file, it is different from how hcl2 parses the original hcl2 code. Reopening as this still causes checkov to fail

**Comment by chrisbloe** [comment]
If it helps, I see the same message with my only tfvars file:

```
terraform scan results:

Passed checks: 3, Failed checks: 2, Skipped checks: 0, Parsing errors: 1

...

Error parsing file C:\***\terraform.tfvars
```

**Comment by robeden** [comment]
@chrisbloe - tfvars problems should be fixed as of a couple days ago (see #865).

**Comment by chrisbloe** [comment]
@robeden - It seems to have resolved all my issues, thanks :)
